### PR TITLE
Add Count items to Tags Overview - v2

### PR DESCRIPTION
**Pull request by pe7er**
Pull Request to **improve** the **Tags** component by adding a **new "Tagged Items" column** that shows a **counter** to offer an insight in how often certain tags are used. The column is sortable so you can easily get an overview about the most or less used tags.

It's PR https://github.com/joomla/joomla-cms/pull/24663 re-done for the current Joomla 4.0-dev.

### Summary of Changes
In the Back-end under Components > Tags you'll see an overview of all created Tags.
However, you don't have any information which tags are actually used, and how often.

### Before PR
tags-before

### Testing Instructions
-  Add some tags to some items (Articles or other items).
-  Check the Components > Tags overview
-  Install the patch
-  Check again the Components > Tags overview

### Expected result
A new column "**Tagged Items**" is available. You can **click** on the "Tagged Items" header **to order** the column. You can select the order also via the "Sort Table By:" dropdown on the top right.

### After PR:
tags-after

### CODE REVIEW

File: administrator/components/com_tags/forms/filter_tags.xml [hunk 1]
```diff
@@ -74,6 +74,8 @@
 			<option value="a.access DESC">JGRID_HEADING_ACCESS_DESC</option>
 			<option value="a.language ASC" requires="multilanguage">JGRID_HEADING_LANGUAGE_ASC</option>
 			<option value="a.language DESC" requires="multilanguage">JGRID_HEADING_LANGUAGE_DESC</option>
+			<option value="a.countTaggedItems ASC">COM_TAGS_COUNT_TAGGED_ITEMS_ASC</option>
+			<option value="a.countTaggedItems DESC">COM_TAGS_COUNT_TAGGED_ITEMS_DESC</option>
```
Comment (SharkyKZ) [inline administrator/components/com_tags/forms/filter_tags.xml]:
```suggestion
			<option value="countTaggedItems ASC">COM_TAGS_COUNT_TAGGED_ITEMS_ASC</option>
			<option value="countTaggedItems DESC">COM_TAGS_COUNT_TAGGED_ITEMS_DESC</option>
```

Comment (pe7er) [reply administrator/components/com_tags/forms/filter_tags.xml]: Thanks!

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 5bf905565b221a327b3fd2a7ca2875c9b45c1ace<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28831">issues.joomla.org/tracker/joomla-cms/28831</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 5bf905565b221a327b3fd2a7ca2875c9b45c1ace<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28831">issues.joomla.org/tracker/joomla-cms/28831</a>.</sub>

**Comment by jwaisner** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28831">issues.joomla.org/tracker/joomla-cms/28831</a>.</sub>

File: administrator/components/com_tags/tmpl/tags/default.php [hunk 2]
```diff
@@ -117,6 +117,9 @@
 								<?php echo HTMLHelper::_('searchtools.sort', 'JGRID_HEADING_LANGUAGE', 'a.language', $this->state->get('list.direction'), $this->state->get('list.ordering')); ?>
 							</th>
 						<?php endif; ?>
+						<th scope="col" style="width:10%" class="d-none d-md-table-cell text-center">
```
Comment (Quy) [inline administrator/components/com_tags/tmpl/tags/default.php]:
```suggestion
						<th scope="col" class="w-10 d-none d-md-table-cell text-center">
```

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**